boards/arm/stm32_min_dev/doc/index.rst

```diff
@@ -30,10 +30,13 @@ port for a specific board. Most of the GPIOs on the STM32 SoC has been exposed
 in the external header with silk screen labels that match the SoC's pin names.
 
 Each board vendor has their own variations in pin mapping on their boards'
-external connectors and placement of components. Many vendors use port PB12 for
+external connectors and placement of components. Many vendors use port PC13 for
 connecting an LED, so only this device is supported by our Zephyr port.
 Additional device support is left for the user to implement.
 
+.. note::
```

Comment:
The content of a note directive needs to be indented.  Also we don't need two links spelled slightly differently:
```
.. note::
   The popular `Blue Pill Board`_ has this LED on PC13. Change the LED
   pin to PB12 in :file:`stm32_min_dev.dts` if you have the `Black Pill Board`_.
```
When I look into stm32_min_dev.dts, I don't see anything obvious to change to "Change the LED pin"?  What do I need to do?

Reply:
Fixing the indentation and the link. 

In line 24 of stm32_min_dev.dts: 
`gpios = <&gpioc 13 GPIO_INT_ACTIVE_HIGH>; ` should be changed to `gpios = <&gpiob 12 GPIO_INT_ACTIVE_HIGH>;` to change the default LED.
Should I mention this in the note?